Client/Frontend/Browser/Toolbars/BottomToolbar/Menu/MenuViewController.swift

```diff
@@ -27,7 +28,16 @@ struct MenuItemHeaderView: View {
                         .padding(6)
                 )
                 .shadow(color: Color.black.opacity(0.1), radius: 1, x: 0, y: 1)
-            Text(verbatim: title)
+            VStack(alignment: .leading, spacing: 5) {
+                Text(verbatim: title)
+                    .padding(.top, subTitle != nil ? 5.0 : 0)
+                if let subTitle = subTitle {
+                    Text(subTitle)
+                        .font(.subheadline)
+                        .foregroundColor(Color(.braveLabel))
+                        .padding(.bottom, 5.0)
```

Comment:
Remove the two paddings, add 1 padding on containing `VStack`: `.padding(.vertical, subtitle != nil ? 5 : 0)`

Reply:
Fix at https://github.com/brave/brave-ios/pull/4424/commits/67ce7646e8e57fc92c7941b2ddbdf30450aaadec